_inc/client/plans/single-product-backup.jsx

```diff
@@ -15,12 +15,33 @@ import PlanPrice from 'components/plans/plan-price';
 
 import './single-product-backup.scss';
 
-export function SingleProductBackup( { products, upgradeLinks } ) {
-	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+export function SingleProductBackup( { plan, products, upgradeLinks } ) {
+	// Don't show the product card for paid plans.
+	if ( ! isEmpty( products ) && ( plan || 'jetpack_free' !== plan ) ) {
```

Comment:
I think this logic isn't right: IF we have products, and any name in the plan name (that includes free plan), don't display the card. Did you mean `plan && 'jetpack_free' !== plan`?

Reply:
@tyxla I had a condition like you suggested before. I was experiencing then an issue that when having a paid plan the placeholder first showed up, then - after the products were loaded - the Backup card content appeared briefly and quickly the whole thing disappeared. Are you able to replicate this issue locally?

Reply:
Yeah, I guess we should update the condition to be more precise and deal with `null` vs `undefined` or something. Haven't really checked how to fix it, but I can totally repro myself